services/ingest/tests/test_api_chat.py

```
@@ -44,7 +44,7 @@ async def test_chat_returns_503_when_mcp_unreachable(client):
 
 @pytest.mark.asyncio
 async def test_chat_streams_sse_response(client):
-    async def fake_run_chat_turn(messages, tools):
+    async def fake_run_chat_turn(messages, tools, org_id):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `fake_run_chat_turn`

(ANN202)